client/app/visualizations/index.js

```diff
@@ -80,11 +86,16 @@ export function registerVisualization(config) {
 -----------------------------------------------------------*/
 
 export function getDefaultVisualization() {
-  return find(registeredVisualizations, visualization => !visualization.isDeprecated);
+  return find(
+    registeredVisualizations,
+    viz => !viz.isDeprecated && viz.isDefault,
+  );
```

Comment:
Now this code assumes that at least one visualization has `isDefault` attribute; not sure if it's right or not, but I'd add a fallback for this case (if no `isDefault` visualization - return first non-deprecated). That's just a suggestion

Reply:
Makes sense. Maybe we should pre-sort the registered visualizations and make the default visualization a constant instead of calling `find()` every time?

This is just a quick hack to avoid Vega showing up as the default editor. If there's any other way of doing it, I'd be happy to avoid adding a new config field.

Reply:
Adding new config field is a good solution, let's keep it. Calling `find` every time is not a problem because ATM it is used only in Edit Visualization dialog and is called there only once (on dialog open).

Reply:
@ktmud I created a separate PR for this feature (getredash/redash#3944) because we needed it right now in few other PRs to fix tests.